README.md

```diff
@@ -58,40 +58,44 @@ Example:
 #include "fast_float/fast_float.h"
 #include <iostream>
 #include <string>
+#include <system_error>
 
 int main() {
-  std::string input = "3.1416 xyz ";
+  const std::string input = "3.1416 xyz ";
   double result;
   auto answer = fast_float::from_chars(input.data(), input.data() + input.size(), result);
-  if (answer.ec != std::errc()) { std::cerr << "parsing failure\n"; return EXIT_FAILURE; }
-  std::cout << "parsed the number " << result << std::endl;
+  if (answer.ec != std::errc()) {
+    std::cerr << "parsing failure\n";
+    return EXIT_FAILURE;
+  }
+  std::cout << "parsed the number " << result << '\n';
   return EXIT_SUCCESS;
 }
 ```
 
-Though the C++17 standard has you do a comparison with `std::errc()` to check whether the conversion worked, you can avoid it by casting the result to a `bool` like so:
+You can simplify the conversion success check by casting the result of `from_chars` to `bool`, as shown below:
```

Comment:
The original text sounds clunky but it makes an important point that this edit removes. The C++17 standard does not include this cast to bool. It's important for users to know that it is a fast_float extension.

The cast to `bool` only became standard in C++26. https://en.cppreference.com/w/cpp/utility/from_chars_result.html

Reply:
Yes, that's easier to comprehend for non-native English speakers. So what about C++20 then?

Reply:
Everything else in that document talks about C++17 `from_chars`, and I think it should be obvious that unless stated otherwise, the same API is present in C++20 and C++23.